pkgs/build-support/mitm-cache/fetch.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, fetchurl
+, runCommand
+, writeText
+}:
+
+{ name ? "deps"
+, data
+, dontFixup ? true
+, ...
+}
+@ attrs:
+
+let
+  data' = builtins.removeAttrs
+    (if builtins.isPath data then lib.importJSON data else data)
+    [ "!version" ];
```

Comment:
Why is `!comment` not handled here?

Reply:
because mitm-cache lockfiles are different from Gradle lockfiles, mitm-cache lockfiles don't have `!comment`